client/src/Resources/ResourceForm.js

```diff
@@ -36,6 +36,9 @@ function ResourceForm() {
         .index()
         .then((response) => {
           setCategories(response.data);
+          if (response.data.length > 0) {
+            setResource((resource) => ({ ...resource, CategoryId: response.data[0].id }));
```

Comment:
Here is the fix- after getting the list of categories back from the server, we simply set the first category's id as the CategoryId in the new resource object. That way, it matches what we see in the dropdown.